enhancements/update/manifest-install-levels.md

```diff
@@ -0,0 +1,196 @@
+
+If a manifest fails any install-phase, valid-manifest guards, the cluster-version operator will complain about the invalid value in a `Failing=True` ClusterVersion condition and not apply any release image manifests.
+That ensures that installs will fail with an understandable explanation, making it very unlikely that accidentally mistyped values make it through CI and into released images.
+
+During the install phase, the cluster-version operator will ensure all manifests in phase *n* are reconciled before attempting to apply any manifests from phase *n*+1.
+Within a phase, manifests will still be applied in parallel without blocking other manifests within that same phase.
+
+Some component manifests are required to satisfy bootstrap completion, which [today is][bootstrap-complete]:
+
+* Production pods for:
+    * `openshift-cluster-version/cluster-version-operator`
+    * `openshift-kube-apiserver/kube-apiserver`
+    * `openshift-kube-controller-manager/kube-controller-manager`
+    * `openshift-kube-scheduler/openshift-kube-scheduler`
+* A bootstraped etcd cluster.
+
+Manifests which are required for bootstrap completion must remain in the default level `0`, ideally by leaving the new annotation unset.
+
+At least one manifest which is fairly slow, ideally always landing after bootstrap removal must remain in the default level `0`, ideally by leaving the new annotation unset.
+For example, the monitoring ClusterOperator always takes a while, especially when it is scheduled to compute nodes.
+FIXME: but when we allow monitoring to be scheduled on the control plane, is it _always_ after bootstrap removal?  On the other hand, all we really need to beat is cluster-bootstrap's manifest-pushing phase, which completes well before its pod-waiting phase.
+
+Manifests which are not required for bootstrap completion, but which are `create-only` defaults for user-facing configuration must be placed in level `1` to avoid racing cluster-bootstrap.
+
+Other manifests should leave the new annotation unset.
+
+### User Stories
+
+#### Custom Network Configuration
+
+Per [rhbz#1832759][rhbz-1832759], CRC sets [a custom MTU][crc-custom-MTU] via [a `create-manifests` flow][crc-create-manifests].
+This currently races [an empty, default Network manifest][extra-manifests-network].
+Once that network manifests grows an `release.openshift.io/install-level: 1` annotation, the user-provided manifest will always be pushed into the cluster.
+
+### Implementation Details/Notes/Constraints
+
+Currently during installation, the cluster-version operator treats manifests as "reconciled" after _attempting_ to push them into the cluster, regardless of whether that push succeeds.
+With this enhancement, it will continue that behavior within an install level, but will then need to wait on successful reconciliation before entering a subsequent install level.
```

Comment:
Will this be a reconciliation? Or just parsing the manifests?

Reply:
"wait on successful reconciliation" means "wait until all manifests within the current install level have successfully reconciled".  There's no step today that only requires parsing the manifests.  Did you have alternative wording you want to float that might make this more clear?